src/python/tests/core/bot/fuzzers/afl/afl_launcher_integration_test.py

```diff
@@ -364,7 +364,7 @@ def _run_with_sudo(command):
       stdin=subprocess.PIPE,
       stdout=subprocess.PIPE,
       stderr=subprocess.STDOUT)
-  popen.communicate(input=SUDO_PASSWORD + '\n')
+  popen.communicate(input=(SUDO_PASSWORD + '\n').encode())
```

Comment:
does utf-8 need to be specified ?

Reply:
This is for input from keyboard, so not really needed. Also on Python 3 the default is utf-8, so leaving it out here to avoid another cleanup in the future.